integrationTests/longTests/executingSCTransactions_test.go

```diff
@@ -392,7 +388,6 @@ func runMultipleRoundsOfTheGame(
 	topUpValue *big.Int,
 	hardCodedScResultingAddress []byte,
 	round uint64,
-	nrShards uint32,
```

Comment:
Why did this parameter dissapear?

Reply:
it is not used. as the propose and sync function uses shardcoordinator.numberOfShards.

Reply:
ok